portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/Publications.testcase

```diff
@@ -727,6 +727,72 @@ definition {
 			changeUser = "Test Test");
 	}
 
+	@description = "This is a use case for LPS-161618 - As a publication user, I would like to track changes to email addresses"
+	@priority = 3
+	test CanTrackChangesToEmailAddresses {
+		task ("Add an Organization and Create a Publication") {
+			JSONOrganization.addOrganization(organizationName = "Organization Name");
+
+			JSONPublications.addPublication(publicationName = "Test Publication");
+
+			JSONPublications.selectPublication(publicationName = "Test Publication");
+		}
+
+		task ("Add email address for the organization") {
+			Organization.openOrganizationsAdmin();
+
+			Organization.gotoEdit(orgName = "Organization Name");
+
+			Navigator.gotoNavTab(navTab = "Contact");
+
+			Navigator.gotoNavNested(navNested = "Contact Information");
+
+			Organization.addContactInformation(
+				address = "organization@liferay.com",
+				contactType = "add-email-address-link");
+		}
+
+		task ("Review changes and publish the publication") {
+			PublicationsNavigator.gotoReviewChanges(
+				gotoPublicationsAdmin = "true",
+				publicationName = "Test Publication");
+
+			AssertElementPresent(
+				key_text = "organization@liferay.com",
+				locator1 = "CP2Utils#ANY_TEXT");
+
+			PublicationsNavigator.gotoPublish(
+				gotoPublicationsAdmin = "true",
+				publicationName = "Test Publication");
+
+			Publications.publishPublication(publicationName = "Test Publication");
+		}
+
+		task ("Revert publication and publish it again") {
+			PublicationsNavigator.openPublicationsHistory();
+
+			Publications.revertPublication(
+				publicationName = "Test Publication",
+				publishNow = "true");
+
+			Publications.publishPublication(publicationName = "Revert \"Test Publication\"");
+		}
+
+		task ("Assert there is no email address in the Organization") {
+			Organization.openOrganizationsAdmin();
+
+			Organization.gotoEdit(orgName = "Organization Name");
+
+			Navigator.gotoNavTab(navTab = "Contact");
+
+			Navigator.gotoNavNested(navNested = "Contact Information");
+
+			AssertElementPresent(
+				key_text = "This organization does not have any additional email addresses.",
+				locator1 = "CP2Utils#ANY_TEXT");
+		}
+	}
+
```

Comment:
You missed the steps [LPS-161618](https://issues.liferay.com/browse/LPS-161618) from step 9 to step 11, can you please add them and resend to me? Thank you!